### ISSUE TITLE
Can you make it work?

### ISSUE THREAD

**Issue by Gosy69** [sopyb/brawlhallaRewardsBot#1]
Can you make it work please?

**Comment by tiefer** [comment]
did it ever work?

**Comment by sopyb** [comment]
Not sure if it works anymore, but it used to work. I didn't use it in a while and forgot about the project until this issue so I might just archive the repository.

I need to see if I have the time to look into what might be broken some time.

**Comment by Gosy69** [comment]
Yes, it would be perfect if you are able to fix it because I really want to use it. Could you please share your Discord username so that I can contact you?

**Comment by sopyb** [comment]
@Gosy69 you can add me @justidktbh
